mmv1/third_party/terraform/website/docs/guides/provider_versions.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # Google Provider Versions
 
-Starting with the `1.19.0` release, there are two versions of the Google
+Starting with the `1.19.0` release, there are two versions of the Google Cloud
```

Comment:
"Google provider" or "Google Cloud Platform provider" is fine, but I'd avoid "Google Cloud provider" as we only support GCP. Google Workspace is part of Google Cloud but not Google Cloud Platform, for example, and is supported through a separate provider.

Reply:
GCP has been disappearing in favour of Google Cloud brand recently, but I added Platform anyway.